components/IscnUploadForm.vue

```diff
@@ -334,5 +516,171 @@ export default class UploadForm extends Vue {
     this.displayImageSrc = this.fileRecords[index].fileData
     this.displayExifInfo = this.fileRecords[index].exifInfo
   }
+
+  onOpenKeplr() {
+    logTrackerEvent(this, 'ISCNCreate', 'OpenKeplr', '', 1);
+    this.isOpenKeplr = true
+    setTimeout(() => {
+      this.isOpenKeplr = false
+    }, 5000)
+  }
+
+  async estimateArweaveFee(): Promise<void> {
+    try {
+      const pricePromises = this.fileRecords.map(record =>
+        estimateBundlrFilePrice({
+          fileSize: record.fileBlob?.size || 0,
+          ipfsHash: record.ipfsHash,
+        }).then(priceResult => ({
+          ...priceResult,
+          ipfsHash: record.ipfsHash,
+        })),
```

Comment:
prefer `await` instead of `.then()` if you don't need to `.catch()` individual errors